manifests/master/install.pp

```diff
@@ -15,13 +17,18 @@
           source => $pupmod::openvox_rpm_path,
         }
       } else {
-        # If the openvox_rpm_path is not provided, install the release package and then the server package
-        package { $pupmod::openvox_repo_path:
+        # If the openvox_release_url is not provided, calculate the release package url
+        $_openvox_release_url = if $facts['os']['name'] == 'Amazon' and $facts['os']['release']['major'] == '2' {
+          "${pupmod::openvox_base_url}/openvox8-release-amazon-2.noarch.rpm "
+        } else {
+          "${pupmod::openvox_base_url}/openvox8-release-el-${facts['os']['release']['major']}.noarch.rpm"
+        }
```

Comment:
This is unconditionally being used (if I'm reading the code correctly) despite what the comment says.

You might want to move all this into the class parameters. Something like this maybe?

```puppet
  Integer         $version     = 8,
  Stdlib::HTTPUrl $release_url = $facts['os']['name'] ? {
    'Amazon' => "${pupmod::openvox_base_url}/openvox${version}-release-${facts['os']['name'].downcase}-${facts['os']['release']['major']}.noarch.rpm",
    default  => "${pupmod::openvox_base_url}/openvox${version}-release-el-${facts['os']['release']['major']}.noarch.rpm",
  },
```

Reply:
I got the variable referenced in the comment wrong, its changed a few times. I changed the comment so it is correct, it will only install the release package and the server from that release repo if the user doesn't specify a specific server package to install, which will let users in airgapped environments to provide the rpm path locally if necessary.